src/main/sensors/barometer.c

```diff
@@ -69,42 +70,48 @@ void pgResetFn_barometerConfig(barometerConfig_t *barometerConfig)
     //
     // 1. If DEFAULT_BARO_xxx is defined, use it.
     // 2. Determine default based on USE_BARO_xxx
-    //   a. Precedence is in the order of popularity; BMP280, MS5611 then BMP085, then
+    //   a. Precedence is in the order of popularity; BMP388, BMP280, MS5611 then BMP085, then
     //   b. If SPI variant is specified, it is likely onboard, so take it.
 
-#if !(defined(DEFAULT_BARO_SPI_BMP280) || defined(DEFAULT_BARO_BMP280) || defined(DEFAULT_BARO_SPI_MS5611) || defined(DEFAULT_BARO_MS5611) || defined(DEFAULT_BARO_BMP085) || defined(DEFAULT_BARO_SPI_LPS) || defined(DEFAULT_BARO_SPI_QMP6988) || defined(DEFAULT_BARO_QMP6988))
-#if defined(USE_BARO_BMP280) || defined(USE_BARO_SPI_BMP280)
-#if defined(USE_BARO_SPI_BMP280)
-#define DEFAULT_BARO_SPI_BMP280
-#else
-#define DEFAULT_BARO_BMP280
-#endif
-#elif defined(USE_BARO_MS5611) || defined(USE_BARO_SPI_MS5611)
-#if defined(USE_BARO_SPI_MS5611)
-#define DEFAULT_BARO_SPI_MS5611
-#else
-#define DEFAULT_BARO_MS5611
-#endif
-#elif defined(USE_BARO_QMP6988) || defined(USE_BARO_SPI_QMP6988)
-#if defined(USE_BARO_SPI_QMP6988)
-#define DEFAULT_BARO_SPI_QMP6988
-#else
-#define DEFAULT_BARO_QMP6988
-#endif
-#elif defined(USE_BARO_SPI_LPS)
-#define DEFAULT_BARO_SPI_LPS
-#elif defined(DEFAULT_BARO_BMP085)
-#define DEFAULT_BARO_BMP085
-#endif
+#if !(defined(DEFAULT_BARO_SPI_BMP388) || defined(DEFAULT_BARO_BMP388) || defined(DEFAULT_BARO_SPI_BMP280) || defined(DEFAULT_BARO_BMP280) || defined(DEFAULT_BARO_SPI_MS5611) || defined(DEFAULT_BARO_MS5611) || defined(DEFAULT_BARO_BMP085) || defined(DEFAULT_BARO_SPI_LPS) || defined(DEFAULT_BARO_SPI_QMP6988) || defined(DEFAULT_BARO_QMP6988))
+    #if defined(USE_BARO_BMP388) || defined(USE_BARO_SPI_BMP388)
+        #if defined(USE_BARO_SPI_BMP388)
+            #define DEFAULT_BARO_SPI_BMP388
+        #else
+            #define DEFAULT_BARO_BMP388
+        #endif
+    #elif defined(USE_BARO_BMP280) || defined(USE_BARO_SPI_BMP280)
+        #if defined(USE_BARO_SPI_BMP280)
+            #define DEFAULT_BARO_SPI_BMP280
+        #else
+            #define DEFAULT_BARO_BMP280
+        #endif
+    #elif defined(USE_BARO_MS5611) || defined(USE_BARO_SPI_MS5611)
+        #if defined(USE_BARO_SPI_MS5611)
+            #define DEFAULT_BARO_SPI_MS5611
+        #else
+            #define DEFAULT_BARO_MS5611
+        #endif
+    #elif defined(USE_BARO_QMP6988) || defined(USE_BARO_SPI_QMP6988)
+        #if defined(USE_BARO_SPI_QMP6988)
+            #define DEFAULT_BARO_SPI_QMP6988
+        #else
+            #define DEFAULT_BARO_QMP6988
+        #endif
+    #elif defined(USE_BARO_SPI_LPS)
+        #define DEFAULT_BARO_SPI_LPS
```

Comment:
Stray whitespace. :space_invader:

Reply:
the unindented version is unreadable by humans (i.e. me), this improves the situation so that I could actually read the code before I modified it to add the BMP388 conditional code.  I can remove the white-space if you feel it's really necessary.

Reply:
Yes please.

Reply:
done